test/byContract/IexecPoco/06_claim.test.ts

```diff
@@ -383,59 +368,34 @@ describe('Poco', async () => {
                 }
             });
 
-            it('Should not initialize and claim array', async function () {
+            it('Should not initialize and claim array if incompatible length of inputs', async function () {
+                const dealId = ethers.utils.hashMessage('dealId');
+                await expect(iexecPoco.initializeAndClaimArray([dealId, dealId], [0])).to.be
+                    .reverted;
+            });
+
+            it('Should not initialize and claim array if one specific fails', async function () {
                 const volume = 2;
                 const { orders } = buildOrders({
                     assets: ordersAssets,
                     requester: requester.address,
                     prices: ordersPrices,
                     volume,
                 });
-                const { dealId, startTime } = await signAndMatchOrders(orders);
-                const taskIndex1 = 0;
-                const taskIndex2 = 1;
-                const task1 = getTaskId(dealId, taskIndex1);
-                const task2 = getTaskId(dealId, taskIndex2);
+                const { dealId, startTime } = await iexecWrapper.signAndMatchOrders(orders);
+                const taskIndex0 = 0;
+                const taskIndex1 = 1;
                 await iexecPoco // Make first task already initialized
                     .connect(scheduler)
```

Comment:
I don't get the diff here with `[task1, task2]` comming from `getTaskId(dealId, taskIndex1)` and now using directly `[taskIndex0,taskIndex1]`

Reply:
Test was wrong, initialize takes dealId & taskIndex